chargeio/infrastructure/ChargeIOError.cs

```diff
@@ -1,23 +1,24 @@
-﻿using System.Collections.Generic;
-using Newtonsoft.Json;
-
-namespace ChargeIO
-{
-	public class ChargeIOError
-	{
-		[JsonProperty("message")]
-		public string Message { get; set; }
-
-		[JsonProperty("code")]
-		public string Code { get; set; }
-
-		[JsonProperty("level")]
-		public string Level { get; set; }
-
-		[JsonProperty("context")]
-		public string Context { get; set; }
-
-    	[JsonProperty("sub_code")]
-		public string SubCode { get; set; }
-	}
+﻿using System;
+using Newtonsoft.Json;
+
+namespace chargeio

```

Comment:
Looking at the current MS docs on naming, this change seems to conflict with their current best practices: https://msdn.microsoft.com/en-us/library/ms229026(v=vs.110).aspx. If we were going to make any change, it seems like it would be to something like "AffiniPay.PaymentGateway", which I would recommend putting off to the later phase of planned re-work and re-naming of the repos.